Parkduksung/ElevenStreetSampleMVVM/app/src/main/java/com/exam/elevenstreet/view/product/ProductViewModel.kt

```diff
@@ -15,9 +15,7 @@ class ProductViewModel(private val productRepository: ProductRepository) {
     private var pageNum = 1
     private var totalCount = 0
 
-    val productItemList = ObservableField<List<ProductItem>>()
-
-    private var _productItemList = mutableListOf<ProductItem>()
+    val productItemList = ObservableField<ProductItem>()
```

Comment:
```suggestion
    val productItem = ObservableField<ProductItem>()
```
단일 item에 맞춰 변수명도 변경이 필요해보여요

Reply:
8c61ae6 수정하였습니다.